setup/quickstart/halyard-local/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: single
+title:  "Halyard on your local machine Quickstart"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+In this quickstart, you will learn the basics of [Halyard](/setup/install/halyard/), Spinnaker's tool for managing your Spinnaker instance.
```

Comment:
This feels like the wrong message - I would switch it to "the basics of getting Spinnaker running for local development"

Reply:
Done.